### DISCUSSION TITLE
Welcome to ThinkCentre-M720Q Discussions!

### DISCUSSION THREAD

**Discussion by revunix** [revunix/ThinkCentre-M720Q#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bradapeter** [comment]
Hi, Could you tell me this efi whether compatitable with M740Q ?  thanks

**Reply by revunix** [reply]
No, it isn't. You have to create a new config for this.
